src/main/webapp/app/entities/event-attendee/event-attendee-sort-button.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Button } from 'reactstrap';
+import { Translate } from 'react-jhipster';
+import { IEventAttendeeSortButtonInfo } from 'app/entities/event-attendee/event-attendee-sort-modal';
+
+export interface IEventAttendeeButtonProps {
+  buttonInfo: IEventAttendeeSortButtonInfo;
+  sort: Function;
```

Comment:
As mention before better to use () => void instead of Function
Can have a enhancement issue for it

Reply:
Ok, will make an issue for it